### PR TITLE
Add github action code

### PR DESCRIPTION
**Pull request by brinxmat**
Adds support for github action and the initial code (I messed up)

### CODE REVIEW

File: external/build.gradle [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    id 'java'
+}
+
+group 'no.unit.nva.model'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
```
Comment (axthosarouris) [inline external/build.gradle]: why are these necessary? doesn't the project inherit those from the parent?

File: external/build.gradle [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
+}
+
+test {
+    useJUnitPlatform()
+}
```
Comment (axthosarouris) [inline external/build.gradle]: ditto

File: internal/build.gradle [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    id 'java'
+}
+
+group 'no.unit.nva.model'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
```
Comment (axthosarouris) [inline internal/build.gradle]: ditto

File: internal/build.gradle [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    def slf4jVersion = '1.7.30'
+
+    implementation project(':external')
+    testImplementation group: 'org.slf4j', name: 'slf4j-api', version: slf4jVersion
+    testImplementation group: 'org.slf4j', name: 'slf4j-simple', version: slf4jVersion
+}
+
+test {
+    useJUnitPlatform()
+}
```
Comment (axthosarouris) [inline internal/build.gradle]: ditto

File: internal/src/main/java/no/unit/nva/model/internal/ResourceDao.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
+package no.unit.nva.model.internal;
```
Comment (axthosarouris) [inline internal/src/main/java/no/unit/nva/model/internal/ResourceDao.java:1]: The DAOs may and (probably will) contain information that is connected to the access patterns of the database. We can start developing it here, but it is likely that it will be moved to the publication-api

**Review by axthosarouris — APPROVED** [review]

File: settings.gradle [hunk 6]
```diff
@@ -0,0 +1,5 @@
+rootProject.name = 'nva-data-model'
+include 'external'
```
Comment (janbj) [inline settings.gradle:2]: Duplicate include?

**Review by janbj — APPROVED** [review]
